nav2_costmap_2d/include/nav2_costmap_2d/costmap_2d_ros.h

```diff
@@ -35,8 +35,8 @@
  * Author: Eitan Marder-Eppstein
  *         David V. Lu!!
  *********************************************************************/
-#ifndef nav2_costmap_2d_nav2_costmap_2d_ROS_H_
-#define nav2_costmap_2d_nav2_costmap_2d_ROS_H_
+#ifndef NAV2_COSTMAP_2D_nav2_costmap_2d_ROS_H_
```

Comment:
I think this should be `NAV2_COSTMAP_2D__COSTMAP_2D_ROS_H_` That is, uppercase filename part, no nav2 prefix on the filename part and double underscores.

Reply:
@SteveMacenski - can you please make this change and we can get this approved

Reply:
done